### PR TITLE
#704 - updated landing page 

### PR DESCRIPTION
**Pull request by aahvocado**
Please test the page in mobile and desktop view sizes

### CODE REVIEW

**Review by sydneywalcoff — CHANGES_REQUESTED** [review]
Hey @aahvocado, the landing page breaks below ~470px;

File: products/statement-generator/public/locales/en/translation.json [hunk 1]
```diff
@@ -75,13 +92,12 @@
     "previewing": "Previewing",
     "introduce_yourself": "Introduce yourself",
     "involvement": "Involvement",
-    "job": "Employment",
```
Comment (sydneywalcoff) [inline products/statement-generator/public/locales/en/translation.json:78]: also I think lines 78 and onward is overriding changes from dev unrelated to what you're updating here.

Comment (aahvocado) [reply products/statement-generator/public/locales/en/translation.json:78]: Good catch!

**Review by sydneywalcoff — CHANGES_REQUESTED** [review]
@aahvocado hey the previous changes look good but the 'complete letter' img breaks the page from 765- ~820px

last change I think!

**Review by sydneywalcoff — APPROVED** [review]

**Review by sydneywalcoff — APPROVED** [review]

**Pull request merged**